Makefile

```diff
@@ -22,6 +23,7 @@ endif # TARGET
 SERVICE := $(or ${SERVICE},quay.io/ocpmetal/assisted-service:latest)
 ISO_CREATION := $(or ${ISO_CREATION},quay.io/ocpmetal/assisted-iso-create:latest)
 DUMMY_IGNITION := $(or ${DUMMY_IGNITION},minikube-local-registry/ignition-dummy-generator:minikube-test)
+LIVE_ISO := $(or ${LIVE_ISO},quay.io/ocpmetal/livecd-iso:rhcos-livecd)
```

Comment:
In case the user will set LIVE_ISO env var that env will be used for the livecd-iso-image tag but that tag isn't used in the build-assisted-iso-generator-image (hard coded: `FROM quay.io/coreos/coreos-installer:release AS installer-image
`)

Reply:
You're right. Reverting back this change since it will likely break ci stuff if I change the assisted-iso-generator dockerfile.